man/systemd.network.xml

```diff
@@ -700,6 +700,17 @@
           <literal>[IPv6PrefixDelegation]</literal> and the <literal>[IPv6Prefix]</literal>
           sections for more configuration options.
           </para></listitem>
+	</varlistentry>
+        <varlistentry>
+          <term><varname>IPv6PDSubnetId=</varname></term>
+          <listitem><para>Configure a specific subnet id from a received prefix delegation to
+          the interface. Allowed values are values from the range 0 to 9223372036854775808
+          (0x8000000000000000, 63 bit) to select a subnet id (as defined in
+          <ulink url="https://tools.ietf.org/html/rfc4291">RFC 4291</ulink>, section 2.5.4)
+	  or -1 (the default) to select an abitrary prefix. This option is only
+	  effective when used together with <varname>IPv6PrefixDelegation=</varname> and the
+	  corresponding configuration on the upstream interface.
```

Comment:
Hmm… I'd personally prefer to specify this in hex notation, and an empty string as the default. Passing this as an integer looks weird to me. WDYT, @ssahani ?

Also note the weird indent, but that's nitpicking ;-)

Reply:
You can just pass hex values prefixed with `0x` in the configuration.

Reply:
It seems that nobody expects decimal notation for this. Please make this hexadecimal base by default. We have appropriate parsers defined, so it's just a matter of switching to the appropriate helper.

Reply:
I changed it to hexadecimal and also adjusted the documentation. Let me know if this looks better now.

I couldn't figure out which trivial helper would do the job here so I added a custom function that also handles the "auto" case (default value).